symbology/create-symbol-styles-from-web-styles/src/main/java/com/esri/samples/create_symbol_styles_from_web_styles/CreateSymbolStylesFromWebStylesSample.java

```diff
@@ -193,34 +187,29 @@ public void start(Stage stage) {
    */
   private void setupLegend() {
 
-    // create a grid pane and set the size, background color and spacing
+    // create a box to show the legend
+    vBox = new VBox();
+    vBox.setMaxSize(220, 300);
+    vBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(255, 255, 255 ,0.5)"), CornerRadii.EMPTY,
+      Insets.EMPTY)));
+    vBox.setPadding(new Insets(10.0));
+    vBox.setAlignment(Pos.TOP_CENTER);
+
```

Comment:
```suggestion
vBox.setSpacing(10);
```
Creates a bit of space between the header and scrollpane